<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>691</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="mainMenu">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>591</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="1">
        <widget class="QPushButton" name="highScoresButton">
         <property name="text">
          <string>Show High Scores</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="playerAmountSpinBox"/>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="cardPairAmountSpinBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Amount of Players:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Memory Match Game (Pairs)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="startGameButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Start Game</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="exitButton">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Amount of Card Pairs:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Please enter player names:</string>
      </property>
     </widget>
     <widget class="QLabel" name="playerNameLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Player 1:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="playerNameLineEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="enterPlayerNameButton">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>51</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="gameBoard">
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>370</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gameOptionGrid">
       <item row="0" column="0">
        <widget class="QPushButton" name="quitMainMenuButton">
         <property name="text">
          <string>Quit to Main Menu</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>591</width>
        <height>301</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gameBoardGrid"/>
     </widget>
     <widget class="QLabel" name="playerTurnLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Player 1's turn:</string>
      </property>
     </widget>
     <widget class="QLabel" name="playerPairCountLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>411</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Player &lt;name&gt; has pairs.</string>
      </property>
     </widget>
     <widget class="QLabel" name="pairMatchLabel">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pair not found</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="highScoresPage">
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>310</y>
        <width>641</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="toMainMenuButton">
         <property name="text">
          <string>Main Menu</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="clearScoresButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Clear High Scores</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>39</y>
        <width>641</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="highScoreGrid"/>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>#. Name - High Score - Lifetime Score</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionInstructions">
   <property name="text">
    <string>Instructions</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>exitButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
